packages/db/index.d.ts

```diff
@@ -74,7 +68,6 @@ export class DBClient {
   getPsaPinRequest (authKey: string, pinRequestId: string) : Promise<PsaPinRequestUpsertOutput>
   listPsaPinRequests (authKey: string, opts?: ListPsaPinRequestOptions ) : Promise<ListPsaPinRequestResults>
   deletePsaPinRequest (pinRequestId: string, authKey: string) : Promise<PsaPinRequestItem>
-  createContent (content: ContentInput, opt?: {updatePinRequests?: boolean}) : Promise<string>
```

Comment:
This function does not exist in the DB client but references pin requests in it's options. I've removed it from here to avoid any confusion.